src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -259,4 +261,13 @@ default GameModeData getGameModeData() {
      */
     void setSleepingIgnored(boolean sleepingIgnored);
 
+    /**
+     * Gets all the {@link KeyBinding}s that are available for this {@link Player},
+     * if the player doesn't have a client that supports custom key bindings this
+     * will always return a empty collection.
+     *
+     * @return The key bindings
+     */
+    Collection<KeyBinding> getKeyBindings();
```

Comment:
Until right now I inferred that key binds were supported server-side like this even on vanilla, but `doesn't have a client that supports...` implies the opposite.

If only (forge?) clients support this call then I think it should be `Optional<Collection<KeyBinding>>` because otherwise it might be assumed that the client supports them but hasn't received any from the server yet.

Thoughts?